controllers/storagecluster/reconcile.go

```diff
@@ -398,11 +398,11 @@ func (r *StorageClusterReconciler) reconcilePhases(
 				&ocsCephObjectStoreUsers{},
 				&ocsCephRGWRoutes{},
 				&ocsStorageClass{},
-				&ocsNoobaaSystem{},
 				&ocsSnapshotClass{},
 				&ocsJobTemplates{},
 				&ocsCephRbdMirrors{},
 				&ocsClusterClaim{},
+				&ocsNoobaaSystem{},
```

Comment:
Looks like NooBaa was already being reconciled towards the end i.e. after its pre-requisites are met.
Since we don't reconcile StorageClient here and can't control when the first StorageClient gets created, how can we be sure that this isn't working just due to a race condition?

Reply:
Will test again and compare the logs with and without this commit